koku/koku/settings.py

```diff
@@ -444,3 +444,5 @@
     DEMO_ACCOUNTS = ENVIRONMENT.json("DEMO_ACCOUNTS", default={})
 except JSONDecodeError:
     pass
+
+OPENSHIFT_DOC_VERSION = "4.5"
```

Comment:
Just a suggestion, but maybe it would be good to make this an importable variable?
```suggestion
OPENSHIFT_DOC_VERSION = ENVIRONMENT.get_value("OPENSHIFT_DOC_VERSION", default="4.5")
```